File: IQCare.CCC/Entities.CCC/Tb/PatientIcf.cs
using Entities.Common;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Entities.CCC.Tb
{
    [Serializable]
    [Table("PatientIcf")]
    public class PatientIcf : BaseEntity
    {
        [Key]
        public int Id { get; set; }
        public int PatientMasterVisitId { get; set; }
        public int PatientId { get; set; }
        public bool Cough { get; set; }
        public bool Fever { get; set; }
        public bool WeightLoss { get; set; }
        public bool NightSweats { get; set; }
    }
}

File: Solutions/IQCare.Core/IQCare/frontend/app/pmtct/_models/hei/PatientIcf.ts
export interface PatientIcf {
    Id?: number;
    PatientMasterVisitId?: number;
    PatientId?: number;
    Cough?: boolean;
    Fever?: boolean;
    WeightLoss?: boolean;
    NightSweats?: boolean;
    DeleteFlag?: boolean;
    CreatedBy?: number;
    CreateDate?: Date;
    OnAntiTbDrugs?: boolean;
    OnIpt?: boolean;
    EverBeenOnIpt?: boolean;
    ContactWithTb?: boolean;
}
